tests/View/Blade/BladeComponentTagCompilerTest.php

```diff
@@ -32,30 +32,30 @@ public function testBasicComponentParsing()
 
         $result = (new ComponentTagCompiler(['alert' => TestAlertComponent::class]))->compileTags('<div><x-alert type="foo" limit="5" @click="foo" required /><x-alert /></div>');
 
-        $this->assertSame("<div> @component('Illuminate\Tests\View\Blade\TestAlertComponent', [])
+        $this->assertSame("<div> @component('Illuminate\Tests\View\Blade\TestAlertComponent', ['attributes' => ['type' => 'foo','limit' => '5','@click' => 'foo','required' => true]])
 <?php \$component->withName('alert'); ?>
 <?php \$component->withAttributes(['type' => 'foo','limit' => '5','@click' => 'foo','required' => true]); ?>\n".
-"@endcomponentClass  @component('Illuminate\Tests\View\Blade\TestAlertComponent', [])
+            "@endcomponentClass  @component('Illuminate\Tests\View\Blade\TestAlertComponent', ['attributes' => []])
```

Comment:
Please revert all of these unnecessary indentations.

Reply:
Done! Not sure what happened here, PHPStorm messed this up a bit.